example-library/src/main/java/uk/gov/gchq/palisade/example/library/config/ExampleUserPrepopulationFactory.java

```diff
@@ -39,7 +39,7 @@
     private String userId = "";
     private Set<String> auths = Collections.emptySet();
     private Set<String> roles = Collections.emptySet();
-    private EnumSet<TrainingCourse> trainingCourses = EnumSet.noneOf(TrainingCourse.class);
+    private Set<TrainingCourse> trainingCourses = EnumSet.noneOf(TrainingCourse.class);
```

Comment:
Would have expected this to stay as an EnumSet.

Reply:
`EnumSet` is a child of `Set` so there shouldn't be any issues with this.